frontend/app_flowy/lib/workspace/presentation/widgets/pop_up_action.dart

```diff
@@ -97,10 +103,7 @@ class ActionCell<T extends ActionItem> extends StatelessWidget {
               children: [
                 if (action.icon != null) action.icon!,
                 HSpace(ActionListSizes.itemHPadding),
-                FlowyText.medium(
-                  action.name,
-                  fontSize: 12,
-                ),
+                FlowyText.medium(action.name, fontSize: 12, color: Colors.black),
```

Comment:
Please use a placeholder instead of black
add a "," after Colors.black (well, the placeholder) so that the formatting is correct